README.md

```diff
@@ -38,7 +38,7 @@ $ python manage.py runserver
 ### Loading FEC's regulations
 
 You will need access to FEC's org in cloud.gov for this.
-Make sure you have run `pip install -r requirements_dev.txt`.
+Make sure you have run `pip install -r requirements.txt && pip install -r requirements_dev.txt`.
```

Comment:
Wouldn't it be better to just _include_ `requirements.txt` in `requirements_dev.txt`?